src/root/views/DrefOperationalUpdateForm/common.tsx

```diff
@@ -3,9 +3,11 @@ import {
   DistrictMini,
 } from "#types/country";
 
-export const ONSET_IMMINENT = 0;
+export const TYPE_IMMINENT = 0;
 export const ONSET_SLOW = 1;
 export const ONSET_SUDDEN = 2;
+export const TYPE_ASSESSMENT = 3;
+export const TYPE_RESPONSE = 4;
```

Comment:
export const ONSET_SLOW = 1;
export const ONSET_SUDDEN = 2;

export const TYPE_IMMINENT = 0;
export const TYPE_ASSESSMENT = 1;
export const TYPE_RESPONSE = 2;

for this take a look at `/dref-options` api